### PR TITLE
Commands: Support sending |selectorhtml| to bot HTML pages

### PR DESCRIPTION
**Pull request by sirDonovan**

### CODE REVIEW

File: server/chat-commands/admin.ts [hunk 1]
```diff
@@ -356,14 +367,17 @@ export const commands: ChatCommands = {
 				pageid: `view-bot-${pageid}`,
 			});
 			context.title = `[${user.name}] ${pageid}`;
-			context.send(content);
+			context.send(content, selector);
```
Comment (Zarel) [inline server/chat-commands/admin.ts:370]:
```suggestion
			if (selector) {
				context.send(`|selectorhtml|${selector}|${content}`);
			} else {
				context.setHTML(content);
			}
```

Comment (sirDonovan) [reply server/chat-commands/admin.ts]: I had to fix a rebase issue so I manually committed this.

**Comment by Zarel** [comment]
> @sirDonovan sirDonovan requested review from AnnikaCodes, HoeenCoder, KrisXV, Marty-D, mia-pi-git and xfix as code owners 4 hours ago

lol wow that bad huh?

File: server/chat-commands/admin.ts [hunk 2]
```diff
@@ -315,11 +315,22 @@ export const commands: Chat.ChatCommands = {
...
+		let targetID: string;
+		let pageid: string;
+		let content: string;
+		let selector: string | undefined;
+		if (cmd === 'changehtmlpageselector') {
+			[targetID, pageid, selector, content] = Utils.splitFirst(target, ',', 3);
+			if (!selector) return this.parse(`/help ${cmd}`);
+		} else {
+			[targetID, pageid, content] = Utils.splitFirst(target, ',', 2);
+		}
```
Comment (Zarel) [inline server/chat-commands/admin.ts:332]:
```suggestion
		let [targetID, pageid, content] = Utils.splitFirst(target, ',', 2);
		let selector: string | undefined;
		if (cmd === 'changehtmlpageselector') {
			[selector, content] = Utils.splitFirst(content, ',');
			if (!selector) return this.parse(`/help ${cmd}`);
		}
```

File: server/chat-commands/admin.ts [hunk 3]
```diff
@@ -354,15 +365,22 @@ export const commands: Chat.ChatCommands = {
...
+			if (selector) {
+				context.send(`|selectorhtml|${selector}|${content}`);
+			} else {
+				context.title = `[${user.name}] ${pageid}`;
+				context.setHTML(content);
+			}
 		}
 
-		this.sendReply(`Sent ${targetUser.name} the bot page ${pageid}.`);
+		this.sendReply(`Sent ${targetUser.name}${(selector ? ` the selector ${selector} on` : '')} the bot page ${pageid}.`);
```
Comment (Zarel) [inline server/chat-commands/admin.ts:376]:
```suggestion
		this.sendReply(`Sent ${targetUser.name}${selector ? ` the selector ${selector} on` : ''} the bot page ${pageid}.`);
```

**Comment by Zarel** [comment]
OK, if you're fine with the new name, I think this is ready.

**Comment by Zarel** [comment]
I'm honestly not a huge fan of the new name but any clearer would make it even longer.

**Pull request merged**